test/Range.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { Range } from '../src/Range';
+
+describe('Range', () => {
+    let range: Range;
+    beforeAll(() => {
+        range = new Range([0, 70], [[2, 2], [4, 5], [10, 15], [32, 40], [50, 50]]).invert().invert();
+    });
```

Comment:
Add tests for `invert`. Right now it could noop and the tests would pass, which is obviously wrong.